foliant.yml

```diff
@@ -46,6 +46,8 @@ chapters:
   - Config Extensions:
     - config/from.md
     - config/slugs.md
+  - По-русски:
```

Comment:
It doesn't work this way, the foliant.yaml file has a strict structure

Reply:
@moigagoo, I guess it's better to set up a new repo for documentation in Russian (and it can be very different, not just a translation), what do you think? Or we can leave it here and use another foliant.yaml for Russian version with the help of [subset extension](https://foliant-docs.github.io/docs/cli/subset/).

Reply:
Nah, I'd rather we have all languages in the same repo. This way, there will be one site with all the docs: foliant-docs.github.io/docs/ru,  foliant-docs.github.io/docs/en, etc.

However, that will break all existing links. If that's not an issue, I think we should go with the one repo approach.

Reply:
@moigagoo, When you talk about all existing links, do you mean external links leading to certain pages of doc-site?

Reply:
@Norrskog yep.

Reply:
In fact, I think I was wrong. It's a bad idea to keep all the docs in one repo, @kvaleev was right about that. MkDocs Material supports localization but doesn't support multilanguage sites. So we better have separate repos for English and Russian docs, separately localized for English and Russian.

Reply:
Since we're not going to add Russian-language pages to this repo, should we now close this pull request?

Reply:
Yeah, I think we should.